tests/raftstore_cases/test_merge.rs

```diff
@@ -449,41 +448,59 @@ fn test_merge_approximate_size_and_keys() {
     let region = pd_client.get_region(b"").unwrap();
 
     cluster.must_split(&region, &middle_key);
+    // make sure split check is invoked so size and keys are updated.
     thread::sleep(Duration::from_secs(1));
 
     let left = pd_client.get_region(b"").unwrap();
     let right = pd_client.get_region(&max_key).unwrap();
-
     assert_ne!(left, right);
+
+    // make sure all peer's approximate size is not None.
+    cluster.must_transfer_leader(right.get_id(), right.get_peers()[0].clone());
```

Comment:
em, you totally sleep too long for this test, can you reduce the sleep time?

Reply:
How to guarantee approximate size is updated?